fix docs example, some clarification (#334)
apollographql/apollo-client-nextjs: packages/experimental-nextjs-app-support/README.md

#### Usage with `useReadQuery`.

> Just like using `useBackgroundQuery` and `useReadQuery` in place of `useSuspenseQuery` (see [the suspense documentation](https://www.apollographql.com/docs/react/data/suspense#avoiding-request-waterfalls)), you can also use `PreloadQuery` in combination with `useReadQuery` in Client Components. Use the render prop notation to get a `QueryRef` that you can pass to your Client Component:

Reviewer: ```suggestion
Just like using `useBackgroundQuery` with `useReadQuery` in place of `useSuspenseQuery` (see [the suspense documentation](https://www.apollographql.com/docs/react/data/suspense#avoiding-request-waterfalls)), you can also use `PreloadQuery` in combination with `useReadQuery` in Client Components. Use the render prop notation to get a `QueryRef` that you can pass to your Client Component:
```

Changing this to `with` makes this sound a bit more like they work together.

Reviewer: I think the thing I'm missing here is the "what" that makes `PreloadQuery` similar to `useBackgroundQuery`. I keep expecting to read something like "Just like using `useBackgroundQuery` with `useReadQuery` in place of `useSuspenseQuery` _to avoid request waterfalls_". Adding a "to x..." statement I think would help make it obvious what the connection between `PreloadQuery` and `useBackgroundQuery` is (obviously feel free to add what you were intending. Your point might not be about avoiding waterfalls).

> Just like using `useBackgroundQuery` and `useReadQuery` in place of `useSuspenseQuery` (see [the suspense documentation](https://www.apollographql.com/docs/react/data/suspense#avoiding-request-waterfalls)), you can also use `PreloadQuery` in combination with `useReadQuery` in Client Components. Use the render prop notation to get a `QueryRef` that you can pass to your Client Component:

Reviewer: ```suggestion
Just like using [`useBackgroundQuery`](https://www.apollographql.com/docs/react/data/suspense#avoiding-request-waterfalls) and `useReadQuery` in place of `useSuspenseQuery`, you can also use `PreloadQuery` in combination with `useReadQuery` in Client Components. Use the render prop notation to get a `QueryRef` that you can pass to your Client Component:
```

Would it make more sense to move this link to `useBackgroundQuery`? I love the link to the documentation, but it feels a touch awkward to interrupt the sentence with the "see x" statement.

Author: Since I'm not linking to the `uBQ` docs, but to the "avoiding waterfalls" section as a generally related concepts, I'd probably keep it separated.

Author: ```suggestion
Just like using `useBackgroundQuery` with `useReadQuery` in place of `useSuspenseQuery` [to avoid request waterfalls](https://www.apollographql.com/docs/react/data/suspense#avoiding-request-waterfalls), you can also use `PreloadQuery` in a similar fashion in combination with `useReadQuery` in Client Components. Use the render prop notation to get a `QueryRef` that you can pass to your Client Component:
```

But I believe this might work.
